guides/md/training_keras_models_on_cloud.md

```diff
@@ -229,55 +258,65 @@ to monitor performance.
 
 
 ```python
-#docs_infra: no_execute
+# docs_infra: no_execute
 model = keras.models.load_model(save_path)
 ```
 
+
 ```python
-#docs_infra: no_execute
+!#docs_infra: no_execute
 !tensorboard dev upload --logdir "gs://keras-examples-jonah/logs/fit" --name "Guide MNIST"
 ```
 
 ---
 ## Large-scale projects
+
 In many cases, your project containing a Keras model may encompass more than one
 Python script, or may involve external data or specific dependencies. TensorFlow
 Cloud is entirely flexible for large-scale deployment, and provides a number of
 intelligent functionalities to aid your projects.
+
 ### Entry points: support for Python scripts and Jupyter notebooks
+
 Your call to the `run()` API won't always be contained inside the same Python script
 as your model training code. For this purpose, we provide an `entry_point` parameter.
 The `entry_point` parameter can be used to specify the Python script or notebook in
 which your model training code lives. When calling `run()` from the same script as
 your model, use the `entry_point` default of `None`.
+
 ### `pip` dependencies
+
 If your project calls on additional `pip` dependencies, it's possible to specify
 the additional required libraries by including a `requirements.txt` file. In this
 file, simply put a list of all the required dependencies and TensorFlow Cloud will
 handle integrating these into your cloud build.
+
 ### Python notebooks
+
 TensorFlow Cloud is also runnable from Python notebooks. Additionally, your specified
 `entry_point` can be a notebook if needed. There are two key differences to keep
 in mind between TensorFlow Cloud on notebooks compared to scripts:
 
 - When calling `run()` from within a notebook, a Cloud Storage bucket must be specified
 for building and storing your Docker image.
 - GCloud authentication happens entirely through your authentication key, without
-project specification. 
-
-An example workflow using TensorFlow Cloud from a notebook
```

Comment:
That last sentence should be separate from the list, I believe

Reply:
That's not how its in the SOT: https://github.com/keras-team/keras-io/blob/master/guides/training_keras_models_on_cloud.py#L273

Reply:
If this needs to be a separate point, I can send a separate PR but I need to get this in to fix the tf.org view.

Reply:
I'm pretty sure it needs to be a separate point, let's fix it in a separate PR to the SoT